Library/CorrelationIdManager.ts

```diff
@@ -49,7 +45,7 @@ class CorrelationIdManager {
                 disableAppInsightsAutoCollection: true
             };
 
-            Logging.info(CorrelationIdManager.TAG, requestOptions);

```

Comment:
Tags are really important in the logs as we can easily identify where the log is coming from, please add them as part of the message

Reply:
added